src/main/scala/net/ruippeixotog/scalascraper/browser/Browser.scala

```diff
@@ -15,6 +15,8 @@ trait Browser {
   def parseFile(path: String, charset: String): Document = parseFile(new File(path), charset)
   def parseFile(path: String): Document = parseFile(new File(path), "UTF-8")
 
+  def parseResource(name: String, charset: String = "UTF-8"): Document = parseFile(new File(getClass.getResource(name).toURI), charset)
```

Comment:
Note that a resource is not always a file. If you are trying to access a resource that is inside a JAR (which doesn't happen when you run your project using SBT, but will always happen once you package your final application), something like this will occur:

```
scala> getClass.getResource("/META-INF/MANIFEST.MF").toURI
res6: java.net.URI = jar:file:/System/Library/Java/Extensions/AppleScriptEngine.jar!/META-INF/MANIFEST.MF

scala> new File(getClass.getResource("/META-INF/MANIFEST.MF").toURI)
java.lang.IllegalArgumentException: URI is not hierarchical
  at java.io.File.<init>(File.java:418)
  ... 42 elided
```

The safest way to read a resource is to use `getClass.getResourceAsStream`. A `parseInputStream` method would be a useful addition to `Browser`! You could implement `parseResource` thereafter using that.